rabbitmq/include/userver/urabbitmq/consumer_base.hpp

```diff
@@ -70,6 +70,12 @@ class ConsumerBase {
   /// It is however guaranteed for message to be requeued if `Process` fails.
   virtual void Process(std::string message) = 0;
 
+  /// @brief Override this method in derived class and implement
```

Comment:
nit: I'd prefer `Override this method ...` -> `You may override this method ...`